### PR TITLE
planner: adjust the schema when using 'explain for' for verbose format

### LINKED ISSUE #26923: index out of range when explain format = 'verbose' for connection id
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
```
use test
show processlist;
explain format = 'verbose' for connection 3;
```

### 2. What did you expect to see? (Required)
```
mysql> show processlist;
+------+------+-----------------+------+---------+------+------------+------------------+
| Id   | User | Host            | db   | Command | Time | State      | Info             |
+------+------+-----------------+------+---------+------+------------+------------------+
|    3 | root | 127.0.0.1:60964 | test | Query   |    0 | autocommit | show processlist |
+------+------+-----------------+------+---------+------+------------+------------------+
1 row in set (0.00 sec)

mysql> explain format = 'verbose' for connection 3;
Query OK, 0 rows affected (0.00 sec)
```

### 3. What did you see instead (Required)
```
mysql> show processlist;
+------+------+-----------------+------+---------+------+------------+------------------+
| Id   | User | Host            | db   | Command | Time | State      | Info             |
+------+------+-----------------+------+---------+------+------------+------------------+
|    3 | root | 127.0.0.1:60964 | test | Query   |    0 | autocommit | show processlist |
+------+------+-----------------+------+---------+------+------------+------------------+
1 row in set (0.00 sec)

mysql> explain format = 'verbose' for connection 3;
ERROR 1105 (HY000): runtime error: index out of range [6] with length 6
```


### 4. What is your TiDB version? (Required)
```
mysql> SELECT tidb_version();
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| tidb_version()                                                                                                                                                                                                                                                                                                            |
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Release Version: v5.2.0-alpha-518-gebb7d7026
Edition: Community
Git Commit Hash: ebb7d70268be38f49db34ebdc115e18eed3e1dff
Git Branch: master
UTC Build Time: 2021-08-05 07:17:38
GoVersion: go1.16.4
Race Enabled: false
TiKV Min Version: v3.0.0-60965b006877ca7234adaced7890d7b029ed1306
Check Table Before Drop: false |
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
1 row in set (0.00 sec)
```

### PR DESCRIPTION
**Pull request by Reminiscent**
### What problem does this PR solve?

Issue Number: close #26923  
related to [PR#23926](https://github.com/pingcap/tidb/pull/23926)

Problem Summary:
Index out of range when explain format = 'verbose' for connection id.

### What is changed and how it works?
Adjust the schema when using 'explain for' for verbose format.

### Check List 

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note 

```release-note
None
```

### CODE REVIEW

File: planner/core/common_plans.go [hunk 1]
```diff
@@ -1275,8 +1279,12 @@ func (e *Explain) prepareOperatorInfo(p Plan, taskType, driverSide, indent strin
 		actRows, analyzeInfo, memoryInfo, diskInfo := getRuntimeInfo(e.ctx, p, nil)
 		row = []string{id, estRows, actRows, taskType, accessObject, analyzeInfo, operatorInfo, memoryInfo, diskInfo}
 	} else if e.RuntimeStatsColl != nil {
+		row = []string{id, estRows}
```
Comment (xuyifangreeneyes) [inline planner/core/common_plans.go:1282]: Do we also need to handle the case `e.Analyze`?

Comment (Reminiscent) [reply planner/core/common_plans.go:1282]: Now `explain analyze` + `format = 'verbose'` seems not to be support yet. So I am not implementation this. What's your opinion? @xuyifangreeneyes

Comment (xuyifangreeneyes) [reply planner/core/common_plans.go:1282]: I am ok with that. Can you leave a comment so that it may be easy for others to know the situation?

**Review by xuyifangreeneyes — APPROVED** [review]

**Review by hanfei1991 — APPROVED** [review]

**Review by xuyifangreeneyes — APPROVED** [review]

**Pull request merged**